tests/safeds/ml/classical/regression/test_ridge_regression.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.ml.classical.regression import RidgeRegression
+
+
+def test_should_raise_if_alpha_is_negative() -> None:
+    with pytest.raises(ValueError, match="alpha must be non-negative"):
+        RidgeRegression(alpha=-1.0)
+
+
+def test_should_warn_if_alpha_is_zero() -> None:
+    with pytest.warns(
+        UserWarning,
+        match=(
+            "Setting alpha to zero makes this model equivalent to LinearRegression. You "
+            "should use LinearRegression instead for better numerical stability."
+        ),
+    ):
+        RidgeRegression(alpha=0.0)
+
+
+def test_should_pass_if_alpha_is_passed_to_fitted_regressor() -> None:
```

Comment:
The idea of the [corresponding test](https://github.com/Safe-DS/Stdlib/blob/b5050b91f17774fa5cf3fc80b51d3ea6c295481f/tests/safeds/ml/classical/regression/test_lasso_regression.py#L22) for `LassoRegression` is different: It checks whether the parameter of the wrapped `scikit-learn` regressor is set correctly.